Adding docs for userroles and permissions on user management dashboard. (#752)
supertokens/docs: v2/emailpassword/custom-ui/init/user-management-dashboard/users-listing-and-details.mdx

---
id: users-listing-and-details
title: "Viewing user list and details"
hide_title: true
---

<!-- COPY SECTION -->
<!-- ./userdashboard/users-listing-and-details.mdx -->
<!-- 1 -->

# Viewing user list and details

> With the user management dashboard you can view the list users and their details. You can also perform different operations on theses users as mentioned below.

Reviewer: ```suggestion
With the user management dashboard you can view the list of users and their details. You can also perform different operations on theses users as mentioned below.
```

Author: image resizing and this done.

> With the user management dashboard you can view the list users and there details that are part of your app and easily you can perform diffrent operations on the users as mentioned below.

Reviewer: ```suggestion
With the user management dashboard you can view the list users and their details. You can also perform different operations on theses users as mentioned below.
```
